server/routes/caseNotes/caseNotesController.ts

```diff
@@ -134,13 +134,20 @@ export default class CaseNotesController {
   ): Promise<void> {
     const { accessToken } = res.locals.user.token
     const { caseNoteId } = req.params
+    const backlinkPageNumber = req.query.backlinkPageNumber as string
```

Comment:
what's the reason for the cast to string? 

if the page number is not included i would expect this to still work (just go back to first page), but i think this would return a 500 ('undefined' isn't a number).

`const { backlinkPageNumber } = req.query` gives you a string or undefined that we can pass through safely

Reply:
also i would probably not ever return 500. if it's not a number just ignore it.

Reply:
but let the presenter handle that

Reply:
I cast as a string as the presenter wouldn't accept it.

Just pulling it straight out of the query params as it is shows the error:

`Argument of type 'string | ParsedQs | string[] | ParsedQs[] | undefined' is not assignable to parameter of type 'string'.   Type 'undefined' is not assignable to type 'string'.`

Reply:
👍 make the presenter take `string | undefined`. casting to string  turns `undefined` into `'undefined'`.